lightning_client_test.go

```diff
@@ -0,0 +1,166 @@
+package lndclient
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"google.golang.org/grpc"
+)
+
+// addInvoiceArg records the args used in a call to mockRPCClient.AddInvoice.
+type addInvoiceArg struct {
+	in   *lnrpc.Invoice
+	opts []grpc.CallOption
+}
+
+// mockRPCClient implements lnrpc.LightningClient with dynamic method implementations and call spying.
```

Comment:
nit: line length here and in several other places. We also try to adhere to the 80 character limit (while counting the tab character as 8 spaces) in this project.

Reply:
This should be wrapped to 80 chars as of [f63e1a1](https://github.com/lightninglabs/lndclient/pull/107/commits/f63e1a18bc056524ad758d10f0e926b4b3084e7c), but feel free to re-open if needed.